packages/app-mobile/root.tsx

```diff
@@ -194,6 +204,10 @@ const appDefaultState = Object.assign({}, defaultState, {
 	route: DEFAULT_ROUTE,
 	noteSelectionEnabled: false,
 	noteSideMenuOptions: null,
+	mobileDataWarning: {
+		isOnMobileData: false,
+		settingOn: false,
```

Comment:
`settingOn` is not needed because you can access the settings directly in the state, under the "settings" key.

Reply:
Neat, I didn't know about that, thanks!